core/src/edu/asu/c3simulator/util/PlayerImp.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * 
+ */
+package edu.asu.c3simulator.util;
+
+import edu.asu.c3simulator.simulation.C3Simulation;
+import edu.asu.c3simulator.simulation.Player;
+import edu.asu.c3simulator.simulation.SocioeconomicStatus;
+
+/**
+ * Implementation of the Player interface
```

Comment:
Need more detail. Add an @see tag. Describe what this implementation is/does. See documentation of LinkedList vs ArrayList for a high-level example